### DISCUSSION TITLE
Curation report week of Jan 25 - Feb 1

### DISCUSSION THREAD

**Discussion by Finterly** [wikipathways/GPML-Repo#42] [Curation reports]
Hi admins,

There were 20 changed pathways this past week with no issues. Mostly changes "replacing an old HMDB identifier with the current one" (Egonw). Among the changes are also 3 new pathways WP5150 (XNauts), WP5157 (XNauts), and WP5158 (Keshav).

Summary 
Recent changes: 20 
New pathways: 3 (WP5150, WP5157, WP5158)
Under construction: 8
Stub pathways: 4
Need approval: 2

Next up: Tina

Fin

**Comment by khanspers** [comment]
Thanks Fin! The author for WP5157 just added a deletion request to that pathway, apparently it is a duplicate of WP5150. I am deleting it now.

**Comment by khanspers** [comment]
Hi all,
I did a quick admin curation round earlier this week to account for a couple of missing weeks. Nothing major to report, many test pathways and new pathways under development from Denise's students. I deleted 3 pathways and approved one (https://www.wikipathways.org/index.php/Pathway:WP5130).

A few stub pathways that have not had recent updates:

PancCanNet pathways that still need work: @mkutmon 
- https://www.wikipathways.org/index.php/Pathway:WP5080
- https://www.wikipathways.org/index.php/Pathway:WP5078 

Adverse Outcome pathway: @marvinm2 
- https://www.wikipathways.org/index.php/Pathway:WP5091

Andra is the regular admin this week.

Kristina
